### DISCUSSION TITLE
Boolean Errors in gdsfactory 8.0.0

### DISCUSSION THREAD

**Discussion by nwasserb** [gdsfactory/gdsfactory#2858] [Q&A]
The First example code here for the Boolean is not working. The result from the Boolean is not printing and I am unsure why?
https://gdsfactory.github.io/gdsfactory/notebooks/04_components_geometry.html

**Comment by fp-matt** [comment]
This looks to me like the issue is that the example notebook hasn't been fully updated to reflect the relatively recent change to the boolean function

I don't remember how the function behaved in prior releases of GDSFactory, but in the current version, there are the `layer1` and `layer2` parameters to specify which layers in `A` and `B`, respectively, to pull the polygons from for the boolean operation. If `layer1` or `layer2` aren't specified, it defaults to trying to pull polygons from the layer specified in the `layer` parameter. 

So, in the example, since `layer=(3, 0)`, and since neither `A` nor `B` have any polygons on those layers, the result of the operation is an empty component.

To fix the example code, you can just provide `layer1=(1, 0)` and `layer2=(2, 0)` to the function's input

I've submitted a PR to fix the docs (#2860)
